apps/epic-web/src/templates/purchased-event-template.tsx

```diff
@@ -147,11 +163,14 @@ const PurchasedEventTemplate = ({
                   <>
                     <p>
                       <Balancer>
-                        Welcome to {product.title}! We're thrilled to have you
-                        join us for this event. Below, you'll find everything
-                        you need to manage your purchase and access related
-                        information. If you have any questions or need
-                        assistance at any point, please don't hesitate to{' '}
+                        <strong>
+                          Welcome to {product.title}! We're thrilled to have you
```

Comment:
here we should probably let them know they’ll receive a zoom link to event via email